css/styles.css

```diff
@@ -18,6 +18,32 @@ Colors:
   color: #e8e8e8;
 }
 
+.divider-lg {
+  width: 23.2rem;
+  height: 4.3rem;
+}
+
+.divider-md {
+  width: 24.7rem;
+  height: 2.4rem;
+}
+
+.margin-bottom-sm {
+  margin-bottom: 1.4rem;
+}
+
+.margin-bottom-md {
+  margin-bottom: 2rem;
+}
+
+.margin-bottom-lg {
+  margin-bottom: 3rem;
+}
+
+.margin-bottom-xl {
+  margin-bottom: 5rem;
+}
```

Comment:
It's atomic css related approach, please move these margins to the classes they belong